build/scripts/build.ps1

```diff
@@ -164,22 +180,40 @@ function Run-MSBuild([string]$projectFilePath, [string]$buildArgs = "", [string]
 # Important to not set $script:bootstrapDir here yet as we're actually in the process of 
 # building the bootstrap.
 function Make-BootstrapBuild() {
-
-    Write-Host "Building Bootstrap compiler"
-    Run-MSBuild "build\Toolset\Toolset.csproj" "/p:UseShippingAssemblyVersion=true /p:InitialDefineConstants=BOOTSTRAP" -logFileName "Bootstrap"
     $dir = Join-Path $binariesDir "Bootstrap"
+    Write-Host "Building Bootstrap compiler"
+    $bootstrapArgs = "/p:UseShippingAssemblyVersion=true /p:InitialDefineConstants=BOOTSTRAP"
     Remove-Item -re $dir -ErrorAction SilentlyContinue
     Create-Directory $dir
-    Move-Item "$configDir\Exes\Toolset\*" $dir
+    if ($buildCoreClr) {
+        $bootstrapFramework = "netcoreapp2.0"
+        $logDir = Join-Path $binariesDir "Logs"
+        Create-Directory $logDir
+        Exec-Console "dotnet" "publish --no-restore src/Compilers/CSharp/csc -o `"$dir/bincore`" --framework $bootstrapFramework $bootstrapArgs -bl:$logDir/BootstrapCsc.binlog"
+        Exec-Console "dotnet" "publish --no-restore src/Compilers/VisualBasic/vbc -o `"$dir/bincore`" --framework $bootstrapFramework $bootstrapArgs -bl:$logDir/BootstrapVbc.binlog"
+        Exec-Console "dotnet" "publish --no-restore src/Compilers/Server/VBCSCompiler -o `"$dir/bincore`" --framework $bootstrapFramework $bootstrapArgs -bl:$logDir/BootstrapVBCSCompiler.binlog"
+        Exec-Console "dotnet" "publish --no-restore src/Compilers/Core/MSBuildTask -o `"$dir`" $bootstrapArgs -bl:$binariesDir/BootstrapMSBuildTask.binlog"
```

Comment:
Why aren't we killing the server here

Reply:
No server to kill. The server can't be used via NuGet package until this is merged :)

Reply:
But once this is merged will need to kill the server. I would add back the kill and just catch the failure condition.

Reply:
Ping on this Q

Reply:
Added Stop-BuildProcesses at the end here, so when this works we'll be all set.

Reply:
@agocke Don't you want to do it at the beginning in case there is detritus from a previous run? Imagine somebody running this script locally to test changes.

Reply:
I believe that's the purpose of the call in the try-finally at the top-level.